### ISSUE TITLE
`conda_forge` activity too fast for its own good?

### ISSUE THREAD

**Issue by CJ-Wright** [regro/rever#117]
It seems that the `conda_forge` activity moves so fast that GH hasn't established the tag/release.

**Comment by hadim** [comment]
Same issue here. Could we add a configurable `time.sleep` call?

**Comment by CJ-Wright** [comment]
Any interest in putting in a PR?

**Comment by hadim** [comment]
I do it manually for the moment. Will make a PR eventually but not before a few weeks.

**Comment by dougalsutherland** [comment]
I just hit a similar problem with pypi:

```
Hashing https://pypi.io/packages/source/a/arxiv-collector/arxiv-collector-0.2.3.tar.gz
activity-error:conda_forge:activity failed with execption:
[...]
urllib.error.HTTPError: HTTP Error 404: Not Found
```

but the file now exists, and running `rever -a conda_forge` afterwards worked.

**Comment by scopatz** [comment]
Yeah, usually I put the conda-forge activity towards the end of the activity list to ensure the source is available.  This could be made smarter,
